TEMPLATE/_mapdata/init.sls

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+---
+{#- Get the `tplroot` from `tpldir` #}
+{%- set tplroot = tpldir.split('/')[0] %}
+{%- from tplroot ~ "/map.jinja" import TEMPLATE as mapdata with context %}
```

Comment:
(`ssf`: Need standard method that can be used across all formulas -- will describe in a subsequent comment).

Reply:
Sure, we need a way to find out which version of `map.jinja` we are using.